packages/volto/src/components/manage/Widgets/__snapshots__/FileWidget.test.jsx.snap

```diff
@@ -208,6 +210,7 @@ exports[`FileWidget renders an empty file widget component 1`] = `
             </div>
             <label
               class="label-file-widget-input"
+              for="field-my-field"
```

Comment:
@Abhishek-17h Modify the tests to have a different id from one test to the other so that we don't have my-field id duplication.
Then have a look at why the label My field points to the same for="field-my-field" 
https://github.com/plone/volto/pull/6592/files#diff-9f7a0caef8d93ad615d7038488525d38b67b32b86153f7441bc973f60b06f713R188